openmoc/plotter.py

```diff
@@ -179,13 +182,13 @@ def plot_segments(track_generator):
   plt.xlim([x.min(), x.max()])
   plt.ylim([y.min(), y.max()])
 
-  title = 'Segments for ' + str(num_azim) + ' angles and ' + str(spacing) + \
-        ' cm spacing'
+  title = 'Segments for ' + str(num_azim) + ' angles, ' + str(spacing) + \
+        ' cm spacing, and ' + str(z[0]) + ' z-level'
 
   plt.title(title)
 
   filename = directory + 'segments-' + str(num_azim) + '-angles-' + \
-      str(spacing) + '-spacing.png'
+      str(spacing) + '-spacing-' + str(z[0]) + '-z-level.png'
```

Comment:
Ditto, if it's agreed that "z-str(z[0])" is more appropriate then use it here as well.